gradio/components/dataset.py

```diff
@@ -103,9 +103,8 @@ def __init__(
             self.headers = [c.label or "" for c in self._components]
         self.samples_per_page = samples_per_page
 
-    @property
-    def skip_api(self):
-        return True
+    def api_info(self) -> dict[str, str]:
```

Comment:
Since we're now including the examples in the API info, we should document the input/output type of `gr.Dataset`